recipes/aospy/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "aospy" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/27/39/d6674839265bbdccbd97291857baddf636e4329e03b6c179bcc62d390970/aospy-0.1.tar.gz
```

Comment:
Please change to `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`